new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java

```diff
@@ -285,20 +335,208 @@ public void beforeEach() {
     }
   }
 
+  /**
+   * Create a WebLogic domain with a defined configmap in configuration/model 
+   * section of the domain resource.
+   * The configmap has multiple sparse WDT model files that define 
+   * a JDBCSystemResource, a JMSSystemResource and a WLDFSystemResource.
+   * Verify all the SystemResource configurations using the rest API call 
+   * using the public nodeport of the administration server.
+   */
+  @Test
+  @Order(1)
+  @DisplayName("Verify the pre-configured SystemResources in a model-in-image domain")
+  public void testMiiCheckSystemResources() {
+    ExecResult result = null;
+    result = checkSystemResourceConfiguration("JDBCSystemResources", "TestDataSource");
```

Comment:
you can replace these lines with single line if you refactor the code, scroll down to check checkSystemResourceConfiguration()

Reply:
done